src/MarkupKind.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace LanguageServerProtocol;
+
+/**
+ * Describes the content type that a client supports in various
+ * result literals like `Hover`, `ParameterInfo` or `CompletionItem`.
+ *
+ * Please note that `MarkupKinds` must not start with a `$`. This kinds
+ * are reserved for internal usage.
+ */
+abstract class MarkupKind
+{
+	/**
+	 * Plain text is supported as a content format
+	 */
```

Comment:
bad indentation

Reply:
Fixed, this was pasted from https://microsoft.github.io/language-server-protocol/specification